README.adoc

```diff
@@ -29,9 +31,28 @@ Updatecli reads a yaml or a go template configuration file, then works into thre
 
 They are many projects out there, which can continuously update your files but they all have an opinionated way of doing it and they often want you to adopt a new platform. Building and distributing software is a difficult thing and good practices constantly evolve. Updatecli was built to work independently wherever you need, and combining whatever workflow you are using as you can see in the following section.
 
+== Installation
+
+Updatecli is a simple Go binary available for Linux, MacOS, Windows. It can be downloaded from the link:https://github.com/updatecli/updatecli/releases[release page]
```

Comment:
```suggestion
Updatecli is a simple Go binary, available for Linux, MacOS, Windows. It can be downloaded from the link:https://github.com/updatecli/updatecli/releases[release page], or installed via link:https://www.updatecli.io/docs/prologue/installation/[different other methods].
```